server/status/monitor.go

```diff
@@ -44,6 +50,7 @@ type NodeStatusMonitor struct {
 	sync.RWMutex
 	stores     map[proto.StoreID]*StoreStatusMonitor
 	nodeID     proto.NodeID
+	desc       proto.NodeDescriptor
```

Comment:
`desc` wasn't being used yet. I've removed `desc`, I'll swap it out with `nodeID` when I need it (likely in my next PR).